### DISCUSSION TITLE
Question Regarding Existing DE Install

### DISCUSSION THREAD

**Discussion by dvub** [prasanthrangan/hyprdots#84] [General]
Hi, I'm looking to start using Hyprland using this setup, but I already have an existing KDE Plasma setup on my desktop, and an XFCE setup on my laptop. Would I be able to install this config for Hyprland without breaking my existing DE's? Thanks.

**Comment by prasanthrangan** [comment]
I have installed this along with my KDE Plasma setup as they share some common packages, it works but not recommended. I would also avoid installing this with an XFCE.

**Reply by dvub** [reply]
Alright, thank you. I did try it alongside my XFCE install without success, but I might have to just do a little more troubleshooting. Thanks though!
